packages/@sanity/code-input/src/CodeInput.js

```diff
@@ -28,15 +31,16 @@ const SUPPORTED_LANGUAGES = [
   {title: 'text', value: 'text'}
 ]
 
-export default class Code extends React.PureComponent {
+class CodeInput extends React.Component {
```

Comment:
Any reason its not a PureComponent anymore?

Its a matter of preference, but I think its clearer to keep the `export default class Code` on the same line. That way you don't have to scroll all the way down if you wondering what this module exports.